hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2026,6 +2026,14 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.ratis.server.read.option</name>
+    <value>DEFAULT</value>
+    <tag>OZONE, OM, RATIS, MANAGEMENT</tag>
+    <description>Select the Ratis server read option.
```

Comment:
Please add the description of the default behavior for Ratis here.

Reply:
Sure.